inotify/adapters.py

```diff
@@ -46,6 +47,16 @@ def __init__(self, paths=[], block_duration=_DEFAULT_EPOLL_BLOCK_DURATION_S):
         for path in paths:
             self.add_watch(path)
 
+    @staticmethod
+    def to_bytes(s):
+        text = unicode if sys.version_info.major == 2 else str
```

Comment:
1) I would prefer that you create another module called "compat.py" and establish the types to use here and in to_unicode() (just below) in that module. 2) Instead of checking for the Python version, just check for whether the `unicode` keyword even exists (using a try-catch, the preferred Pythonic approach).